src/components/ControlBar/EditBar.js

```diff
@@ -172,7 +210,7 @@ const EditBar = props => {
         return <Redirect to={redirectUrl} />
     }
 
-    const { updateAccess } = props
+    const { newDashboard, updateAccess } = props
```

Comment:
It's a little strange to destructure just these two props at this point. I would say remove this line and use the `props.propName` notation instead. Especially as the use of `newDashboard` is above this line.

Reply:
fixed

Reply:
fixed